assets/data/cards_pt-br.json

```diff
@@ -550,7 +550,34 @@
         "Ferramenta",
         "Versionamento"
       ],
-      "description": "O GitHub é uma plataforma de hospedagem de código-fonte baseada na web que oferece uma variedade de serviços relacionados ao controle de versão e colaboração no desenvolvimento de software (é usado em conjunto com o Git). Ele é amplamente utilizado por pessoas desenvolvedoras de todo o mundo para armazenar, colaborar, gerenciar e controlar o código-fonte de projetos de software."
+      "description": "O GitHub é uma plataforma de hospedagem de código-fonte baseada na web que oferece uma variedade de serviços relacionados ao controle de versão e colaboração no desenvolvimento de software (é usado em conjunto com o Git). Ele é amplamente utilizado por pessoas desenvolvedoras de todo o mundo para armazenar, colaborar, gerenciar e controlar o código-fonte de projetos de software. Fornece ferramentas de CI/CD (integração contínua e entrega contínua) com o GitHub Actions"
+    },
+    {
+      "title": "GitLab",
+      "tags": [
+        "Conceito",
+        "Ferramenta",
+        "Versionamento"
+      ],
+      "description": "Assim como GitHub, o GitLab também é uma plataforma de hospedagem de código-fonte, mas além de versionar projetos como o GitHub, oferece ferramentas integradas de CI/CD (integração contínua e entrega contínua) diretamente."
```

Comment:
> mas além de versionar projetos como o GitHub, oferece ferramentas integradas de CI/CD

Como o GitHub e o [BitBucket](https://www.atlassian.com/br/software/bitbucket/features/pipelines) também fazem isso, não sei se essa parte ainda faz sentido. Talvez valha falar alguma outra coisa que ela tem de diferente, mas a única coisa de diferente que eu sei que tem é que vários códigos open source mudaram pro GitLab quando a Microsoft comprou o GitHub.

https://about.gitlab.com/blog/2018/05/31/welcome-gnome-to-gitlab/

https://about.gitlab.com/blog/2020/06/29/welcome-kde/